src/FfmpegInit.cs

```diff
@@ -61,6 +63,7 @@ public static void UseSpecificLogCallback(Boolean storeLogs = true)
                 ffmpeg.av_log_format_line(p0, level, format, vl, lineBuffer, lineSize, &printPrefix);
                 var line = Marshal.PtrToStringAnsi((IntPtr)lineBuffer);
                 //Console.Write(line);
+                Debug.WriteLine("\n!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n"+line+"\n!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n");

```

Comment:
Does this need to be in the released package?

Reply:
definitely this should be removed in the released package